aggregator_api/src/lib.rs

```diff
@@ -83,7 +83,7 @@ pub fn aggregator_api_handler<C: Clock>(ds: Arc<Datastore<C>>, cfg: Config) -> i
         State(ds),
         State(Arc::new(cfg)),
         // Metrics.
-        metrics("janus_aggregator_api").with_route(|conn| conn.route().map(ToString::to_string)),
+        metrics("janus_aggregator").with_route(|conn| conn.route().map(ToString::to_string)),
```

Comment:
Clarification: is it still possible to differentiate metrics coming from the aggregator api vs aggregator?

Reply:
The only metric names that the two crates both use are "http.server.duration" et. al., but those can be differentiated via the `http_route` label. Otherwise, our custom metrics were partitioned between different scopes.